lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -166,10 +166,17 @@
                         courseID = this.courseID;
                     self.getTopic(topicID).done(function(topic) {
                         self.getTeam(teamID).done(function(team) {
-                            var view = new TeamProfileView({
-                                courseID: courseID,
-                                model: team
-                            });
+                            var readOnly = !(
+                                self.$el.data('privileged') ||
+                                    _.any(team.attributes.membership, function (membership) {
+                                        return membership.user.id === self.$el.data('username');
```

Comment:
`id` is the property with the username, which is coming from Ben's `CollapsedReferenceSerializer`. This needs to change, though, since passing `expand=user` in the request does return `username` instead of `id`.